practice-one/src/pages/Analytics/Analytics.jsx

```diff
@@ -1,7 +1,59 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+import "./Analytics.css";
+import { Button, Expand, CustomerItem } from "@components";
+import { BUTTON_VARIANTS } from "@constants/buttons";
+import { plusIcon } from "@assets/images";
+import { EXPAND_TITLES } from "@data";
 
 const Analytics = () => {
-  return <h1>Analytic</h1>;
+  const [customers, setCustomers] = useState([]);
+
+  useEffect(() => {
+    // Sử dụng fetch để fetch dữ liệu từ API
+    fetch("https://64e70a09b0fd9648b78f3e0a.mockapi.io/customers")
+      .then((response) => response.json())
+      .then((data) => {
+        // Lưu trữ dữ liệu từ API vào state
+        setCustomers(data);
+      })
+      .catch((error) => {
+        console.error("Error fetching data:", error);
+      });
+  }, []);
+  return (
+    <>
+      <div className="analytics__header">
+        <h1 className="analytics__title">Customer List</h1>
+        <Button variant={BUTTON_VARIANTS.SECONDARY} icon={plusIcon} />
+      </div>
+
+      <div className="customer__table">
+        {/* Start sort title */}
+        <div className="customer__sort">
+          {EXPAND_TITLES.map((EXPAND_TITLE) => (
+            <div className="sort__item" key={EXPAND_TITLE.id}>
+              <Expand expandName={EXPAND_TITLE.title} />
+            </div>
+          ))}
+        </div>
+
+        {/* Start customer list */}
+        <ul className="customer__list">
+          {customers.map((customer) => (
```

Comment:
what will show is the custom is empty?

Reply:
I added a message will show when the list is empty 
![image](https://github.com/datdragon777/React-Training/assets/74806826/7d4bf01e-f27e-484d-b061-8358da357084)

Reply:
style center for empty customers: _Customers is empty!_